src/Revise.jl

```diff
@@ -1115,7 +1115,7 @@ end
 
 function setup_atom(atommod::Module)::Nothing
     handlers = getfield(atommod, :handlers)
-    for x in ["eval", "evalall", "evalrepl"]
+    for x in ["eval", "evalall", "evalshow"]
         old = handlers[x]
```

Comment:
For safety, can you change this to
```julia
        if haskey(handlers, "x")
            old = handlers[x]
            ...
```
Since `"evalrepl"` still exists, is there any reason to exclude it from the list?

Reply:
Sure.

> Since "evalrepl" still exists, is there any reason to exclude it from the list?

The reasoning was that `evalrepl` isn't used in recent Juno versions.
But someone might use Revise in the old Juno versions, so this safety check would be better.

***

Well, why didn't you included `evalshow` within the previous implementation ?
Any reasoning behind that ?

Reply:
I think I just forgot to update Revise when I added the `evalshow` handler :)